docs/ONCALL.adoc

```diff
@@ -0,0 +1,13 @@
+== Dates Oncall:
+
+=== Summary/Notes:
+
+* Use this section to put high level context that may be helpful to the next person on-call
+
+=== Continuing Issues
+
+* Put issues that weren't fully resolved here, include a link and a risk summary
+
+=== Issues
+
+* Put each issue responded to here. There should be a link to the ticket
```

Comment:
I would like to see the following metrics/breakdowns of the issues from the week:

* *Action summary breakdown:* this would categorize each issue into one of the following buckets:
  * no action taken, on-going maintenance (expired window) - link to CCR
  * no action taken, on-going maintenance (no window set) - link to CCR
  * no action taken, threshold adjustment needed - link to change of threshold in code or monitoring system
  * no action taken, check is not reliable/needs updating - link to ticket to fix check script
  * action taken - link to ticket with investigative comments/notes

* Sleep state summary: this needs to be specified by the on-call responder as to whether the alert came in while they were asleep or awake.

There are a couple more that we might add eventually but these will be a good start to understanding our on-call issues.